core/src/components/button/button.ionic.scss

```
@@ -0,0 +1,234 @@
+  
+  font-size: #{$button-ionic-large-font-size};
+}
+
+/* Extra Large Button */
+:host(.button-xlarge) {
+  --padding-top: #{$button-ionic-xlarge-padding-top};
+  --padding-end: #{$button-ionic-xlarge-padding-end};
+
+  min-height: #{$button-ionic-xlarge-min-height};
+
+  font-size: #{$button-ionic-xlarge-font-size};
+}
+
+// Button with Icons
+// -------------------------------------------------------------------------------
+
+/* Button containing only an icon */
+::slotted(ion-icon[slot=start]),
+::slotted(ion-icon[slot=end]),
+::slotted(ion-icon[slot=icon-only]) {
+  font-size: 1em;
+}
+
+/* Button extra small */
+:host(.button-has-icon-only.button-xsmall) {
+  --padding-end: #{$button-has-icon-only-padding-end-xsmall};
+}
+
+/* Button small */
+:host(.button-has-icon-only.button-small) {
+  --padding-end: #{$button-has-icon-only-padding-end-small};
+}
+
+/* Default */
+:host(.button-has-icon-only) {
+  --padding-end: #{$button-has-icon-only-padding-end};
+}
+
+/* Button large */
+:host(.button-has-icon-only.button-large) {
+  --padding-end: #{$button-has-icon-only-padding-end-large};
+}
+
+/* Button extra large */
+:host(.button-has-icon-only.button-xlarge) {
+  --padding-end: #{$button-has-icon-only-padding-end-xlarge};
+}
+
+// Button Shapes
+// -------------------------------------------------------------------------------
+
+/* Rectangular */
+:host(.button-rectangular) {
+  --border-radius: #{$button-ionic-default-rectangular-border};
+}
+
+/* Round */
+:host(.button-round) {
+  --border-radius: #{$button-ionic-default-round-border};
+}
+
+// Button States
+// -------------------------------------------------------------------------------
+
+/* Activated */
+:host(.ion-activated) {
+  --background: #{ion-color(primary, shade)};
+}
+
+:host(.button-outline.ion-activated),
+:host(.button-clear.ion-activated) {
+  --color: #{ion-color(primary, base)};
+  --background: #e3e3e3;
+}
+
+/* Focused */
+// Only show the focus ring when the button is focused
+:host(.ion-focused){
+  --overflow: visible;
+  
+  .button-native::after {
+    /* stylelint-disable-next-line property-disallowed-list */
+    border-radius: inherit;
+  }
+}
+
+.button-focus-ring {
+  --focus-ring-color: #9ec4fd;
+  --focus-ring-width: 2px;
+  
+  @include position(-4px, -4px, -4px, -4px);
+
+  position: absolute;
+
+  transition: border-color 0.3s;
+
+  border: var(--focus-ring-width) solid var(--focus-ring-color);
+
+  /* stylelint-disable-next-line property-disallowed-list */
+  border-radius: inherit;
+
+  content: "";
+  box-sizing: border-box;
+}
+
+// Outline
+:host(.button-outline) {
+  --border-width: #{$button-ionic-outline-border-width};
+  --border-style: #{$button-ionic-outline-border-style};
+  --background-activated: #{ion-color(primary, base)};
+  --background-focused: #{ion-color(primary, base)};
+  --background-hover: transparent;
+  --background-focused-opacity: .1;
+  --color-activated: #{ion-color(primary, contrast)};
+}
```

```suggestion
:host(.button-outline) {
  --border-width: #{$button-ionic-outline-border-width};
  --border-style: #{$button-ionic-outline-border-style};
  --background-activated: #e3e3e3;
  --background-activated-opacity: 1;
  --background-focused: transparent;
  --background-hover: transparent;
  --background-focused-opacity: .1;
  --color-activated: #{ion-color(primary, base)};
}

:host(.button-outline.ion-focused) {
  --border-width: 0;
}
```

I might be wrong but I think these are the values for outline to match the Figma.